src/features/embalm/stepContent/components/GenerateRecipientPDF.tsx

```diff
@@ -89,6 +89,7 @@ export function GenerateRecipientPDF() {
               as="span"
               variant="link"
               onClick={downloadRecipientPDF}
+              cursor="pointer"
```

Comment:
This is more nuanced change. The button should always have the pointer but since I added the `as='span'` then that removes the cursor. And should be noted that the span doesn't do what I want it to. So in this case I would remove the as='span'. (the span was there to get the those 2 sentences on the same line, but it didn't work and i should have removed it)